src/cluster_linearize.h

```diff
@@ -711,8 +721,11 @@ class SearchCandidateFinder
              *  pot_feerate. It starts off equal to inc. */
             auto pot = inc;
             if (!inc.feerate.IsEmpty()) {
-                // Add entries to pot.
-                for (auto pos : und) {
+                // Add entries to pot. We iterate over all undecided transactions whose feerate is
+                // higher than best. While undecided transactions of lower feerate may improve pot
+                // still, if they do, the resulting pot feerate cannot possibly exceed best's (and
```

Comment:
fbb988a6128b859b204299357cb6bb4133f83e25, are there some extra words?
```suggestion
                // higher than best. While undecided transactions of lower feerate may improve pot,
                // the resulting pot feerate cannot possibly exceed best's (and
```